cobra/test/test_io/test_json.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+"""Test functionalities of json.py"""
+
+from __future__ import absolute_import
+
+from os.path import join
+from warnings import warn
+
+import pytest
+from cobra import io
+from cobra.test.test_io.conftest import compare_models
+
+try:
```

Comment:
Take a look at [this section](https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency) and the pytest skipif functionality in general for a better approach to running or not running certain tests. (I know it's done this way in existing tests but we are trying to improve, right? :wink: )

Reply:
pytest sure has a lot of tricks! :P